a11y/FrontendAccessibilitySpec.scala

```diff
@@ -0,0 +1,184 @@
+      "weightOrVolume" -> optional(ignored(BigDecimal(0))),
+      "country" -> text,
+      "originCountry" -> optional(text),
+      "currency" -> ignored(""),
+      "cost" -> ignored(BigDecimal(0)),
+      "isVatPaid" -> optional(boolean),
+      "isExcisePaid" -> optional(boolean),
+      "isCustomPaid" -> optional(boolean),
+      "hasEvidence" -> optional(boolean)
+    )(TobaccoDto.apply)(TobaccoDto.unapply)
+  )
+  implicit val arbTobaccoInput: Arbitrary[Form[TobaccoDto]] = fixed(tobaccoForm)
+  // Another limitation of the framework is that it can generate Arbitrary[T] but not Arbitrary[T[_]],
+  // so any nested types (like a Play `Form[]`) must similarly be provided by wrapping
+  // a concrete value using `fixed()`.  Usually, you'll have a value you can use somewhere else
+  // in your codebase - either in your production code or another test.
+  // Note - these values are declared as `implicit` to simplify calls to `render()` below
+  // e.g implicit val arbReportProblemPage: Arbitrary[Form[ReportProblemForm]] = fixed(reportProblemForm)
+
+  // This is the package where the page templates are located in your service
+  val viewPackageName = "views.html"
+
+  // This is the layout class or classes which are injected into all full pages in your service.
+  // This might be `HmrcLayout` or some custom class(es) that your service uses as base page templates.
+  val layoutClasses = Seq(classOf[views.html.templates.GovukLayoutWrapper])
+
+  // this partial function wires up the generic render() functions with arbitrary instances of the correct types.
+  // Important: there's a known issue with intellij incorrectly displaying warnings here, you should be able to ignore these for now.
+  override def renderViewByClass: PartialFunction[Any, Html] = {
+    case errorTemplate: errorTemplate => render(errorTemplate)
+    case timeOut: timeOut => render(timeOut)
+    // alcohol
+    case alcohol_input: alcohol_input => render(alcohol_input)
+    // amendments
+    case declaration_not_found: declaration_not_found => render(declaration_not_found)
+    case declaration_retrieval: declaration_retrieval => render(declaration_retrieval)
+    case no_further_amendment: no_further_amendment => render(no_further_amendment)
+    case pending_payment: pending_payment => render(pending_payment)
+    case previous_declaration: previous_declaration => render(previous_declaration)
+    // declaration
+    case declare_your_goods: declare_your_goods => render(declare_your_goods)
+    case enter_your_details: enter_your_details => render(enter_your_details)
+    case zero_to_declare_your_goods: zero_to_declare_your_goods => render(zero_to_declare_your_goods)
+    // errors
+    case purchase_price_out_of_bounds: purchase_price_out_of_bounds => render(purchase_price_out_of_bounds)
+    // other_goods
+     case other_goods_input: other_goods_input => render(other_goods_input)
+    // purchased_products
+    case done: done => render(done)
```

Comment:
remove and dashboard missing here, please add. Thanks

Reply:
<img width="582" alt="Screenshot 2023-05-15 at 12 55 23" src="https://github.com/hmrc/bc-passengers-frontend/assets/63714619/d85bb6df-f5e8-4c73-9ea4-7f565f8fd8c0">

<img width="578" alt="Screenshot 2023-05-15 at 12 56 45" src="https://github.com/hmrc/bc-passengers-frontend/assets/63714619/5090df43-0402-4ded-bde1-f95da995bf83">